### PR TITLE
Chronometer in Android

### LINKED ISSUE #1457: Android-Kotlin : 09.26 Chronometer in Android (Kotlin)
<hr>

## Description 📜

* Give a step-by-step tutorial of Chronometer in Android (Kotlin).
*  Required Documentation or Video or Audio.

<hr>

## Domain of Contribution 📊

- [x] Android Dev (Flutter, Java, Kotlin)

<hr>

<hr>

<hr>

## Please Read it Carefully Before Contributing
* Changes should be made in **Android_Development_With_Kotlin/** directory under **Android_Development_With_Kotlin** branch.
* Inside **Android_Development_With_Kotlin/** directory creates a directory and names it according to the topic ( for this issue **09. Dynamic User Interface**)
* Inside **09. Dynamic User Interface/** directory add your task with subtopic number and name and make changes accordingly.
* You have to push your PR to the **Android_Development_With_Kotlin** branch only.
* You can take help to the references but don't simply try to copy it. Just explain in your own word.
* This issue is for **GWOC'21** contributors only.
* You need to push in a single commit. If there exists more than one commit then you need to **squash** your commits into one commit.
* Will assign you on a first come first serve basis.
* Please try to complete your assigned task within 7 days. If you need some more time then, please inform mentors and supervisors.
* If you have two previously assigned issues on **Android-Kotlin** pending then please first complete them.
> Supervisor @rahulsain

## Happy Contributing. Best of Luck.💖

### PR DESCRIPTION
**Pull request by Parth-001-git**
<hr>

## Description 📜

In this repo. you will get to know about chronometer with very well know language kotlin. 
Fixes #1457

<hr>

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [x] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>
Chronometer img

<hr>

### CODE REVIEW

**Comment by rahulsain** [comment]
Attach some screenshots, make the code separate from the texts. Rename your file and its folder structure.

Check merged PR's for some example.

**Comment by Parth-001-git** [comment]
@rahulsain Please review .

File: Chronometer in Android.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        android:text="Start"
+        android:onClick="startStopChronometer"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/chronometer"
+        />
+</android.support.constraint.ConstraintLayout>'
+
+**Activity**
+'import android.os.Bundle;
```
Comment (rahulsain) [inline Chronometer in Android.md:61]: enclose it in the code format

File: Chronometer in Android.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+            isStart = false;
+            ((Button)view).setText("Start");
+        }else{
+            chronometer.setBase(SystemClock.elapsedRealtime());
+            chronometer.start();
+            isStart = true;
+            ((Button)view).setText("Stop");
+        }
+    }
+} '
```
Comment (rahulsain) [inline Chronometer in Android.md:100]: till here

File: Chronometer in Android.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+**AndroidManifest.xml file**
+
+'<?xml version="1.0" encoding="utf-8"?>
```
Comment (rahulsain) [inline Chronometer in Android.md:104]: code format start from here

File: Chronometer in Android.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+    <activity android:name=".MainActivity">
+        <intent-filter>
+            <action android:name="android.intent.action.MAIN" />
+ 
+            <category android:name="android.intent.category.LAUNCHER" />
+        </intent-filter>
+    </activity>
+</application>
+ 
+</manifest>'
```
Comment (rahulsain) [inline Chronometer in Android.md:124]: end here

File: Chronometer in Android.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+|android:gravity|	                       It specifies the gravity of the view like center, top, bottom, etc|
+|------|------| 
+|android:format|	                        It defines the format of the string to be displayed.|
+|------|------| 
+|android:countDown|	         It defines whether the chronometer will count up or count down.|
+
+**Layout**
+All codes related to  Chronometer with text appearance and background style settings are listed  below.
+
+'<?xml version="1.0" encoding="utf-8"?>
```
Comment (rahulsain) [inline Chronometer in Android.md:29]: code format start here

File: Chronometer in Android.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+        android:layout_height="wrap_content"
+        android:layout_margin="19dp"
+        style="@style/Widget.AppCompat.Button.Colored"
+        android:text="Start"
+        android:onClick="startStopChronometer"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/chronometer"
+        />
+</android.support.constraint.ConstraintLayout>'
```
Comment (rahulsain) [inline Chronometer in Android.md:58]: end here

**Review by rahulsain — CHANGES_REQUESTED** [review]

**Comment by Parth-001-git** [comment]
@rahulsain  Please review.

**Comment by rahulsain** [comment]
> Attach some screenshots, make the code separate from the texts. Rename your file and its folder structure.
> 
> Check merged PR's for some example.

Please do this before asking for review

**Comment by sitamadex11** [comment]
@Parth-001-git your documentation is not structured. Please try to give some spaces between codes and use proper heading and subheadings with bullets and numbering. please give some screenshots of your XML file and final output. Also, you can add a gif of your final output.

Also your change your file structure as supervisor mentioned.

**Pull request closed without merging**